src/test/java/resources/batchStatus.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Batches API validation test
+
+  Scenario: BS-001,002,003 Batches API with invalid offset, limit and sort value
+    Given I have tenant as "paymentbb2"
+    When I call the Batches API with invalid offset, limit and sort value expecting status of 400
```

Comment:
you can add more details about the values sent so that this file gives complete idea about the testcase